src/app/shared/mock-auth.service.ts

```diff
@@ -0,0 +1,8 @@
LTkxNTUtNjNjYjYwYzVmNDdlIiwiYWxsb3dlZC1vcmlnaW5zIjpbIioiXSwicmVhbG1fYWNjZXNzIjp7InJvbGVzIjpbInVtYV9hdXRob3JpemF0aW9uIl19LCJyZXNvdXJjZV9hY2Nlc3MiOnsiYnJva2VyIjp7InJvbGVzIjpbInJlYWQtdG9rZW4iXX0sImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyIsInZpZXctcHJvZmlsZSJdfX0sImF1dGhvcml6YXRpb24iOnsicGVybWlzc2lvbnMiOlt7InNjb3BlcyI6WyJyZWFkOnNwYWNlIiwiYWRtaW46c3BhY2UiXSwicmVzb3VyY2Vfc2V0X2lkIjoiYzk3MGI4MzQtOTRjNi00OTYwLWE0MGUtNmY2ZGM0MTQ0YjQ5IiwicmVzb3VyY2Vfc2V0X25hbWUiOiI0ZGY5MzkyYi05Mzc3LTQ2NDYtYjZkOC0yNTlkM2Y4NGQ4MTcifSx7InNjb3BlcyI6WyJyZWFkOnNwYWNlIiwiYWRtaW46c3BhY2UiXSwicmVzb3VyY2Vfc2V0X2lkIjoiMzc0MmM0YzYtNDc4ZS00MmQzLTgzNTQtNGQ4M2ZjM2Y5NjhhIiwicmVzb3VyY2Vfc2V0X25hbWUiOiJiY2RkODQzNi00MjVhLTQzOGItODY4MS01ZjliZjZkYTYwNGUifSx7InNjb3BlcyI6WyJyZWFkOnNwYWNlIiwiYWRtaW46c3BhY2UiXSwicmVzb3VyY2Vfc2V0X2lkIjoiMmIyMDM0MGYtNTNkMi00ZDRkLTgwMDgtZTQ2MmQ2YzYzOGM4IiwicmVzb3VyY2Vfc2V0X25hbWUiOiI0ZDk3OTU2Ny1mMmVhLTRiM2YtOWE1Mi05NDU1NGE5NTU3M2MifSx7InNjb3BlcyI6WyJyZWFkOnNwYWNlIiwiYWRtaW46c3BhY2UiXSwicmVzb3VyY2Vfc2V0X2lkIjoiNDc5ZWY2OWEtMDczNy00NmUyLTliY2QtZWE1MWNkNTQ3NzM4IiwicmVzb3VyY2Vfc2V0X25hbWUiOiI1MWM3NzQ3MS0yZDUyLTRjMDYtOTY1YS1jZjYwNWUwMjc4YzQifSx7InNjb3BlcyI6WyJyZWFkOnNwYWNlIiwiYWRtaW46c3BhY2UiXSwicmVzb3VyY2Vfc2V0X2lkIjoiZTVlNDExMzAtY2VmNS00YWM2LWJkN2ItMzUyZDI5MjM2ZDExIiwicmVzb3VyY2Vfc2V0X25hbWUiOiJmMTVlY2MxZC04NWRlLTQzYTQtOTk3MC1jZGE3NWRkZDYyOWMifSx7InNjb3BlcyI6WyJyZWFkOnNwYWNlIiwiYWRtaW46c3BhY2UiXSwicmVzb3VyY2Vfc2V0X2lkIjoiN2Y4ZjQ4YTMtMDRiOC00YmFmLTg2YmQtYmE3MzBjMzEwYzc5IiwicmVzb3VyY2Vfc2V0X25hbWUiOiI4NTZmNDA2NC0zNDYwLTQyNDItYWZjYy01MDBkNWMxYjdkMzIifSx7InNjb3BlcyI6WyJyZWFkOnNwYWNlIiwiYWRtaW46c3BhY2UiXSwicmVzb3VyY2Vfc2V0X2lkIjoiOWRkZmE4Y2ItYWRiNi00NGI4LThmMDAtZTMwMmM1MWFiY2NiIiwicmVzb3VyY2Vfc2V0X25hbWUiOiJmYjk4MmUyNi01MzhlLTRlNjEtYmU4My1jOWM0OTY0YTQ4NmIifSx7InNjb3BlcyI6WyJyZWFkOnNwYWNlIiwiYWRtaW46c3BhY2UiXSwicmVzb3VyY2Vfc2V0X2lkIjoiZTRkYjVlMzUtNjNiZC00MjkxLTk0ZmItNjBhM2UxMzU2OGM0IiwicmVzb3VyY2Vfc2V0X25hbWUiOiJmODYzZWE5Yi1iMzI0LTQwNjctYjQ5Ny1lN2FmNzdjY2EwMmYifSx7InNjb3BlcyI6WyJyZWFkOnNwYWNlIiwiYWRtaW46c3BhY2UiXSwicmVzb3VyY2Vfc2V0X2lkIjoiMTNkNTY4YmMtYTA4NC00ZGY2LWFkOWQtODg3MTg2OTc2NzM3IiwicmVzb3VyY2Vfc2V0X25hbWUiOiI1YWY2YmUyYy1lYjNkLTQzZDMtYTVlYi04MjRlOWE4N2JjMWIifSx7InNjb3BlcyI6WyJyZWFkOnNwYWNlIiwiYWRtaW46c3BhY2UiXSwicmVzb3VyY2Vfc2V0X2lkIjoiOTk0MTQ5NmMtZmI2Mi00ODQzLWE4YzItOTIwYmZlNDY5YWEzIiwicmVzb3VyY2Vfc2V0X25hbWUiOiI3MTVmNDI1OS1lZTNjLTQ0OTItOTVhMC1jNjM5M2FjZmFlZmIifSx7InNjb3BlcyI6WyJyZWFkOnNwYWNlIiwiYWRtaW46c3BhY2UiXSwicmVzb3VyY2Vfc2V0X2lkIjoiODUzN2M3YWMtNmYwMy00YTMyLWE5OGUtM2EyYjQ5OTllMGY0IiwicmVzb3VyY2Vfc2V0X25hbWUiOiI1YzFkNzMzNC02NjRmLTQyMDktYjEwMy1hNzBjZmQ0NTQxNmEifV19LCJjb21wYW55IjoiUmVkaGF0In0.BlMxjuHkfRuF8rTdtmkcurQ8BcaBekCXn2eu1FfUMtc7fOGm75Lm3unjnswNt5PNtCS_Qfi95jG4yNmsZ2h3AMfScJu5CLzYCG8fg-bzfYKNR4HOFmvUUvRHEW7Fgo8QsLzM7P0Kw2AOKcdt1E2jMduRtkaCAteMrALjfIm-1A2TTv7-Q_0I_WqHfRwfOj73WSNkP2vFpjd7jUQTKjR4KLx0mMU5-jDNGRIRKfloyfjbW42xm8pwV3M9Xbo10s12DQl0dxYuZHbQBL7gULke7827Yra2CmYAUEKqTgMf8O45iyiLbD9qyI8S_aDNz8R23krHH-Z_j1R2q_MG73OpGw';
```

Comment:
@invincibleJai @humaton Ya I understand that. But this is going to be only for standalone and it makes it hard for all the developers to add the same in environment and for that we have to
 save this token somewhere just for that. Just a thought. Otherwise no other problem I have doing this